src/KeePass.Win.Services/HelloProvider.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Windows.Security.Credentials;
+using Windows.Security.Credentials.UI;
+using Windows.UI.Xaml;
+
+namespace KeePass.Win.Services
+{
+    public class HelloProvider
```

Comment:
Can you implement this as an ICredentialProvider? The pattern I'm using is composing credential providers within each other. See the autofac module for how this is being done.